index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Product Landing Page</title>
+        <meta charset="UTF-8" name="viewport" content="width=device, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header id="header">
+            <img id="header-img" src="360_F_339931799_lxc97IIbZ3ZeXYNPg3y15MIr8nCf1g8c.jpg" alt="Company Logo"><br>
+            <h1>Chawanzi & Dire Inc.</h1><br>
+            <nav id="nav-bar" >
+                <ul>
+                    <li class="nav-link"><a href="#">Home</a></li>
+                    <li class="nav-link"><a href="#">Gallery</a></li>
+                    <li class="nav-link"><a href="#">About Us</a></li>
+                </ul><br>
+                <video id ="video" width="" height="" controls>
+                    <source src="" type="">
+                </video>
+            </nav>
+        </header><br>
+
+        <form id="form" action= "">
+            <label for="Email">Email</label>
+            <input type ="text" input id = "email" placeholder="Enter your email" required>
+            <input id = "submit" type="submit" value="Submit" action="https://www.freecodecamp.com/email-submit">
+        </form><br>
+
+        <meta name="viewport" content="widith=device-width,initial-scale=1.0" charset="UTF-8">
+    </head>
+    <body>
+        <header id="header">
+            <img id="header-img" src="" alt="Company Logo" >
+            <nav id="nav-bar" >
+                <ul>
+                    <li class="nav-link"><a href="#"></a></li>
+                    <li class="nav-link"><a href="#"></a></li>
+                    <li class="nav-link"><a href="#"></a>5</li>
+                </ul>
+                <video id = "video" width="" height="" controls>
+                    <source src="" type=""></video>
+            </nav>
+
+            </header>
+        <form id="form" action= "">
+            <label for="Email">Email</label>
+            <input type ="text" input id = "email" placeholder="Enter your email" required>
+            <input id = "submit" type="submit" value="Submit" a href = "https://www.freecodecamp.com/email-submit">
+        </form>
+    </body>
```

Comment:
Please replace the body tag with a section tag. The body tag is meant to contain all the other elements in your codebase.